flink-end-to-end-tests/test-scripts/test_streaming_classloader.sh

```diff
@@ -28,6 +28,9 @@ sed -i -e 's/classloader.resolve-order: .*//' "$FLINK_DIR/conf/flink-conf.yaml"
 sed -i -e 's/classloader.parent-first-patterns: .*//' $FLINK_DIR/conf/flink-conf.yaml
 echo "classloader.resolve-order: parent-first" >> "$FLINK_DIR/conf/flink-conf.yaml"
 
+echo "Moving fake LibPackage.jar from end-to-end tests to lib/"
+cp ${END_TO_END_DIR}/flink-parent-child-classloading-test-lib-package/target/LibPackage.jar ${FLINK_DIR}/lib/
```

Comment:
Undo the move in a cleanup trap similar to `test_streaming_sql.sh`.

Reply:
adding that!